openreview/venue/group.py

```diff
@@ -277,6 +286,9 @@ def create_venue_group(self):
         if self.venue.request_form_id:
             content['request_form_id'] = { 'value': self.venue.request_form_id }
 
+        if self.venue.request_form_invitation:
+            content['request_form_invitation'] = { 'value': self.venue.request_form_invitation }
```

Comment:
@xkopenreview is this keyword exists then the venue is running on the new workflow configuration